ydb/docs/en/core/changelog-cli.md

```diff
@@ -2,13 +2,41 @@
 
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.8.0 {#2-8-0}
+
+Released on January 12, 2024. To update to version **2.8.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+**Features:**
+
+* Added configuration management commands for the cluster `ydb admin config` and `ydb admin volatile-config`.
+
+* Added the `generic` type of requests in the [ydb table query execute](reference/ydb-cli/table-query-execute.md) command, allowing to perform [DDL](https://en.wikipedia.org/wiki/Data_Definition_Language) and [DML](https://en.wikipedia.org/wiki/Data_Manipulation_Language) operations, with arbitrarily-sized results and with support for [MVCC](concepts/mvcc.md). The command uses an experimental API, compatibility is not guaranteed.
```

Comment:
it's also better to move all these experimental features to the bottom of the feature list